src/gocept/selenium/scripts/converthtmltests.py

```diff
@@ -174,10 +172,10 @@ def main(args=None):
         encoding = 'utf-8'
 
     if len(methods) == 0:
-        print(u"No file was generated !")
+        print("No file was generated !")
         return
     if options.verbose:
-        print(u"Generating [%s]" % target)
+        print("Generating [%s]" % target)
```

Comment:
Here and at some other places, f-strings seem also possible. But I guess it is not mandatory to catch all places of use, when `pyupgrade` did not found it.

Reply:
I was too lazy to try to find all the places where f-strings are possible.
This can be done when the code is touched for the next time.